apps/api/src/app/widgets/e2e/remove-all-messages.e2e.ts

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import { MessageRepository, NotificationTemplateEntity, SubscriberRepository } from '@novu/dal';
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+import { ChannelTypeEnum } from '@novu/shared';
+
+describe('Remove all messages - /widgets/messages (DELETE)', function () {
+  const messageRepository = new MessageRepository();
+  let session: UserSession;
+  let template: NotificationTemplateEntity;
+  let subscriberId: string;
+  let subscriberToken: string;
+  let subscriberProfile: {
+    _id: string;
+  } | null = null;
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+    subscriberId = SubscriberRepository.createObjectId();
+
+    template = await session.createTemplate({
+      noFeedId: true,
+    });
+
+    const { body } = await session.testAgent
+      .post('/v1/widgets/session/initialize')
+      .send({
+        applicationIdentifier: session.environment.identifier,
+        subscriberId,
+        firstName: 'Test',
+        lastName: 'User',
+        email: 'test@example.com',
+      })
+      .expect(201);
+
+    const { token, profile } = body.data;
+
+    subscriberToken = token;
+    subscriberProfile = profile;
+  });
+
+  it('should remove all messages', async function () {
```

Comment:
What do you think about also adding a test case for the delete of a specific feed?

Reply:
Yeah. I agree. It will increase coverage. I will add

Reply:
Added